code/game/dna/genes/vg_powers.dm

```diff
@@ -239,9 +239,11 @@
 	action_icon_state = "genetic_view"
 
 /obj/effect/proc_holder/spell/targeted/remoteview/choose_targets(mob/user = usr)
-	var/list/targets = living_mob_list
+	var/list/targets = list()
 	var/list/remoteviewers = new /list()
-	for(var/mob/M in targets)
+	for(var/mob/M in living_mob_list)
+		if(PSY_RESIST in M.mutations)
+			continue
 		if(REMOTE_VIEW in M.mutations)
 			remoteviewers += M
 	if(!remoteviewers.len || remoteviewers.len == 1)
```

Comment:
You'll probably want to remove the `len == 1` check here. It's meant to fail if the only valid target is the caster, but now the caster may not be a valid target at all.